server/statistics/store.go

```diff
@@ -265,3 +297,6 @@ func (r *RollingStoreStats) GetInstantLoad(k StoreStatKind) float64 {
 	}
 	return 0
 }
+
+// SlowStoreStats is a cached statistics for the slow store.
+type SlowStoreStats struct{}
```

Comment:
why not use interface{}

Reply:
IMO, although `SlowStoreStatInfo` is an empty `struct{}` now for implementing `set` now, I think it might collect more detailed infos about `slow` in the later development. So, I just keep it as `struct {}`.